tutorials/getting-started/old/setting-up-your-environment.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
Why these changes to something in "/old" that appear to duplicate the new getting started article?